### DISCUSSION TITLE
How to request the history of a PR to be removed

### DISCUSSION THREAD

**Discussion by miniman** [community/community#35713] [Other Feature Feedback, Questions, & Ideas]
Hi all,

I'm very not a junior birdman and just accidently put up a PR against the parent repo of a fork instead of my own repo; i.e., I specified the wrong remote. I am flogging myself with a cat-o-nine tails in shame and guilt but that doesn't get me anywhere when it comes to requesting the history of the PR to be removed. This isn't a Copyright Infringement on behalf of the repo owner; whom I greatly respect and wish zero harm or penalty; this was my sleep-deprivation caused eff-up. As well, this is inadvertantly a private information infringement infraction, but entirely self-induced. The damage is confined to a single sub-directory of a PR; the remainder of the PR includes some minor patches to the originally forked code.

If repo is named `acme/foo/` and I put up a PR for `yadda/bar/ to merge with `acme/foo/` by mistake, how can I delete that PR, or certainly a sub-directory of that PR like `yadda/bar/buzz/`?

Sincerly,
--chagrined

**Comment by airtower-luna** [comment]
Very tricky. *Within* your own repository you can remove or edit the relevant commits, and then ask support to clear them from the cache, see: [Removing sensitive data from a repository](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository)

The tricky thing here is that making a PR also creates a pr ref in the target repository, so what's on your branch will be easily available there even if you delete the branch or your whole fork. You'll have to ask support what they can do, I assume the policy on deleting data from someone else's repository will be very restrictive (and it certainly needs to be).
